js/main.js

```diff
@@ -356,8 +356,12 @@ function updatedb(){
       var s = (warn>1) ? "s" : "";
       document.getElementById("overallstate").innerHTML="<font class='text-danger'><i class='bi bi-exclamation-circle'></i>&nbsp;"+warn+" critical value"+s+" detected!</font>";
       warnuser(warn);
+      $(document).attr('title', $(document).attr('title').replace("Loading...", "WARNING"));
+      $(document).attr('title', $(document).attr('title').replace("OK", "WARNING"));
     }else{
       document.getElementById("overallstate").innerHTML="<font class='text-success'><i class='bi bi-check2-circle'></i>&nbsp;System runs normally</font>";
+      $(document).attr('title', $(document).attr('title').replace("Loading...", "OK"));
+      $(document).attr('title', $(document).attr('title').replace("WARNING", "OK"));
```

Comment:
As a short remark: Couldn't we save a line of code by simply calling this before ajax call:
`
$(document).attr('title','<?php system("hostname");?> - Loading...');
`
This way the ajax call will be reflected in document title and if unsuccessful a user could easily spot, that in one browser tab the connection to RPi was lost.

Btw I think you should use `$(el).prop()` instead of `.attr()` - please check:
https://stackoverflow.com/questions/5874652/prop-vs-attr

Reply:
Loading basically is there all the time. the code above just changes this loading to Warning or OK. Line 360 and 364 are for each global update in case there is a change of the status.

Reply:
I'm pretty sure you did not get my point - "Loading..." is only there on inital page load - after that it does never appear again! Therefore I would like to call
`$(document).attr('title','<?php system("hostname");?> - Loading...');`
before starting ajax call. This shows user when update happens and most importantly: you can see if it timed out etc...

Reply:
Should be fine now